extensions/shared/get-allowed-mime-types.js

```diff
@@ -48,3 +48,35 @@ export function pickFileExtensionsFromMimeTypes( mimeTypesObject ) {
 export default function getAllowedMimeTypesBySite() {
 	return get( getJetpackData(), [ 'allowedMimeTypes' ], [] );
 }
+
+/**
+ * Check if the given file is a video.
+ *
+ * @param   {string|object} file - File to check.
+ * @param   {string}        type - File type used to check the file.
+ * @returns {boolean}       True if file type matches with the given type. Otherwise, False.
+ */
+export function checkFileType( file, type ) {
```

Comment:
How would you feel about calling it `isVideoFile`?

Reply:
`isVideoFile` was the name before to this refactoring when we needed to check only video files. Now we are checking different types:

```es6
if ( checkFileType( file, 'video' ) {
  // is a video!
} else if ( checkFileType( file, 'audio' ) {
  // is an audio!
} else {
  // who knows
}
```

I did this refactoring just because we need to check if the file dropped in the `core/audio` block is an audio file.